### PR TITLE
Add thrift echo Crossdock test

### PR DESCRIPTION
**Pull request by kriskowal**
r @yarpc/yarpc

### CODE REVIEW

**Comment by blampe** [comment]
ok

File: crossdock.md [hunk 1]
```diff
@@ -35,17 +57,63 @@ Dimensions: `server`, `transport`
...
+        "status": "passed",
+        "output": "made http thrift request to Echo::echo - got token: 34bdchk"
+    }
 ]
 ```
 
-Dimensions: `server`, `transport`, each Thrift `procedure` in [ThriftTest.thrift](ThrifTest.thrift)
+Dimensions: `server`, `transport`
+
+thrift-coverage
```
Comment (breerly) [inline crossdock.md]: Let's rename this to `thrift-gauntlet` - that's the name we've been using for this in discussions.

**Comment by breerly** [comment]
:+1:

**Comment by abhinav** [comment]
:+1:

File: crossdock.md (hunk 1 above)
Comment (kriskowal) [inline crossdock.md]: Great name :ok_hand:

**Pull request merged**